raft/README.md

```diff
@@ -5,6 +5,14 @@ The state machine is kept in sync through the use of a replicated log.
 For more details on Raft, see "In Search of an Understandable Consensus Algorithm"
 (https://ramcloud.stanford.edu/raft.pdf) by Diego Ongaro and John Ousterhout.
 
+This Raft library is stable and feature completed. It is **the most widely used** (as of 2016) Raft library that serves tens of thousands clusters each day. It powers distributed systems like etcd, Kubernetes, Docker Swarm, Cloud Foundry Diego, CockroachDB, TiDB, Project Calico, Flannel and more.
+
+Most Raft implementations has a monolithic design, which includes storage, messaging serialization, transport. This Raft library takes a very different approach by only implementing the core algorithm part of raft. The design philosophy of this raft library is minimalism, flexibility, determinism, and performance.
```

Comment:
Most Raft implementations have a monolithic design, including storage handling, messaging serialization, and network transport. This library instead follows a minimalistic design philosophy by only implementing the core raft algorithm. This minimalism buys flexibility, determinism, and performance.